kernel/data-pipeline/dialect/mysql/src/test/java/org/apache/shardingsphere/data/pipeline/mysql/sqlbuilder/MySQLPipelineSQLBuilderTest.java

```diff
@@ -61,4 +61,20 @@ private DataRecord mockDataRecord(final String tableName) {
         result.addColumn(new Column("c3", "", true, false));
         return result;
     }
+    
+    @Test
+    public void assertQuoteKeyword() {
+        String tableName = "CASCADE";
+        String actualCountSql = sqlBuilder.buildCountSQL(null, tableName);
+        assertThat(actualCountSql, is(String.format("SELECT COUNT(*) FROM %s", sqlBuilder.quote(tableName))));
+        actualCountSql = sqlBuilder.buildCountSQL(null, tableName.toLowerCase());
+        assertThat(actualCountSql, is(String.format("SELECT COUNT(*) FROM %s", sqlBuilder.quote(tableName.toLowerCase()))));
+    }
+    
+    @Test
+    public void assertQuoteNormal() {
```

Comment:
1, Method name `assertQuoteNormal` could be improved, there's no `quoted` in SQL.

2, Could we simplify `expectedDropSQL`, `String.format` is not necessary.

And also in other implementations.